anomalib/data/utils/image.py

```diff
@@ -66,3 +69,23 @@ def read_image(path: Union[str, Path]) -> np.ndarray:
     image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)
 
     return image
+
+
+def pad_nextpow2(batch: Tensor) -> Tensor:
+    """Compute required padding from input size and return padded images.
+
+    Finds the largest dimension and computes a square image to pass into the model. In case the image dimension
```

Comment:
Maybe mention in the description that the function pads the images so that both sides are a power of 2.